src/cancerhotspots/index.js

```diff
@@ -272,7 +274,8 @@ const uploadFile = async ({ filename, conn, errorLogPrefix }) => {
     const previousLoad = new Set();
     logger.info('load previous statements');
     const statements = await conn.getRecords({
-        filters: convertRecordToQueryFilters({ neighbors: 0, returnProperties: 'sourceId', source: rid(source) }),
```

Comment:
why is convertRecordToQueryFilters no longer used?

Reply:
This was leftover from a previous version of the GraphKB module. neighbours is no longer a valid argument at the filters level